aQute.libg/src/aQute/lib/json/Encoder.java

```diff
@@ -131,7 +132,7 @@ public void close() throws IOException {
 		}
 	}
 
-	void encode(Object object, Type type, Map<Object, Type> visited) throws Exception {
+	public void encode(Object object, Type type, Map<Object, Type> visited) throws Exception {
```

Comment:
public because someone may would create an custom Handler and needs to use the encode method.

Example:
https://github.com/bndtools/bnd/pull/4655/files#diff-716c31f069558da5d52fcd64b5e0d5edee400820378678b9584fb2944a59f763R91